Documentation/ApiOverview/PageTypes/CreateNewPageType.rst

```diff
@@ -16,58 +16,83 @@ the directions below to the end:
 ..  versionchanged:: 12.0
     A new :php:`PageDoktypeRegistry` was introduced replacing the
     :php:`$GLOBALS['PAGES_TYPES']` array. Use the version selector to look up
-    the syntax in the corresponding documentation version for older TYPO3 versions.
+    the syntax in the corresponding documentation version for older
+    TYPO3 versions.
 
-The first step is to add the new page type to the
-:php:`\TYPO3\CMS\Core\DataHandling\PageDoktypeRegistry`. Then you need to add
-the icon chosen for the new page type and allow users to drag and drop the new
-page type to the page tree.
+..  rst-class:: bignums
 
-The new page type is added to the :php:`PageDoktypeRegistry` in
-:file:`ext_tables.php`:
+1.  Add new page type to :php:`PageDoktypeRegistry`
 
-..  literalinclude:: _ext_tables.php
```

Comment:
should this file be deleted now it is not used anymore?

Reply:
It's not removed. It's moved to line 35